### ISSUE TITLE
issue with Poor audio rate% (audio poor percentage) once uploaded to BI service

### ISSUE THREAD

**Issue by AndrewMarsh24** [MattWade-MS/QER#7]
Might not be an issue but seeing that when in BI desktop, the Poor audio rate% (audio poor percentage) is correct in desktop, i upload to BI service and the refresh the data in it changes from a % to a decimal and is then skewing the figures.

i have completely deleted the dataset in BI service and the dashboard and reuploaded and done the same refresh and the issue reappears. its as if the data model its pulling from has changed slightly

**Comment by stasfeelin** [comment]
We see the same inconsistency.
Here are some TS steps we tried:

- We confirmed that by using same credential in Power BI Service, we still unable to see the correct data.
- We tried to upload to another workspace to avoid being affected by cache, but still see the incorrect data.
- We confirmed that the visual using same filter in Power BI Service and Power BI Desktop
- We confirmed that the visual is using “Direct Query” mode to retrieve data, that means the data is returned by data source in real time when view the report
- We created a new visual contain the row counter of this table, and confirmed the service & Desktop got the same row count, but the data inside it is different.
- We tried to convert the table to “Import” Mode, that means we can get the data then store in data model, but in this test. We saw the connector did not return any data for this table.

We have an MSFT support ticket opened as well to investigate.
